newsfragments/1572.misc.rst

```diff
@@ -0,0 +1 @@
+Added README file for macOS packaging process and packaging scripts
```

Comment:
Newsfragment end up in the changelog that is designed to be read by end users.
So "Add support for macOS" would be better here